lib/cli.mjs

```diff
@@ -556,11 +558,15 @@ export default async function main(argv) {
 
 	return standalone(options)
 		.then((linted) => {
-			if (!linted.output) {
+			if (!linted.output && !linted.code) {
 				return;
 			}
 
-			process.stdout.write(linted.output);
+			if (linted.code) {
+				process.stdout.write(linted.code);
+			}
+
+			process.stderr.write(linted.output);
```

Comment:
[suggest] How about writing conditionally like this to avoid outputting an empty string?

```suggestion
			if (linted.output) {
				process.stderr.write(linted.output);
			}
```

Reply:
Good catch! Done here: 8d7bfdaada3d713dab564add646547e9026901ca.